cime/src/share/util/shr_mem_mod.F90

```diff
@@ -91,8 +91,8 @@ subroutine shr_mem_getusage(r_msize,r_mrss,prt)
     !---------------------------------------------------
 
     ierr = GPTLget_memusage (msize, mrss, mshare, mtext, mdatastack)
-    r_msize = msize*mb_blk
-    r_mrss  = mrss*mb_blk
+    r_msize = msize / 1024_shr_kind_r8
+    r_mrss  = mrss  / 1024_shr_kind_r8
```

Comment:
@mt5555 just a note for your review, fortran now relies on mem-usage from C sys-calls.